Source/Core/DiscIO/VolumeWiiCrypted.cpp

```diff
@@ -68,34 +67,40 @@ bool CVolumeWiiCrypted::Read(u64 _ReadOffset, u64 _Length, u8* _pBuffer, bool de
 	{
 		static unsigned char IV[16];
```

Comment:
You can probably get rid of this and pass "m_pBuffer + 0x3d0" directly to aes_crypt_cbc() (it will be overwritten but so what). At the least it doesn't look like it needs to be static.